docs/release-notes/8.1.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[release-notes-header-8.1.0]]
+== 8.1
+
+[discrete]
+[[release-notes-8.1.0]]
+=== 8.1.0
+
+[discrete]
+[[breaking-changes-8.1.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are re-used in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+There are no breaking changes in 8.1.0.
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.1.0]]
+==== Features
+* Improves copy for the privilege check on the Endpoint page ({pull}124118[#124118]).
+* Introduces a new *Host risk classification* column in the All hosts table on the *Hosts* page. In addition, a new *Host by risk* tab has been added to the *Hosts* page and host detail pages. From the *Host by risk* tab, you can access an explanation of how a host’s risk is calculated and scored ({pull}122980[#122980], {pull}122586[#122586], {pull}122018[#122018], {pull}121075[#121075], {pull}120487[#120487], {pull}119734[#119734]).
+* Introduces the ability to  bulk edit rule index patterns and tags ({pull}122635[#122635]).
+* Expands Endpoint per-policy artifact assignment to include endpoint event filters and host isolation IP exceptions ({pull}121879[#121879], {pull}121632[#121632]).
+* Adds the rule execution UUID field to alerts. In addition, the `kibana.alert.rule.execution.uuid` is now part of the alert data schema and can be found in the field browser for the Alert's table.({pull}113058[#113058]).
+* Introduces case metrics that summarize alert information and response times ({pull}121336[#121336]).
+
+[discrete]
+[[bug-fixes-8.1.0]]
+==== Bug fixes and enhancements
+* Improves the performance of indicator match rules ({pull}123882[#123882], {pull}123677[#123677]).
+* Changes the default indicator index query of custom and prebuilt indicator match rules to `@timestamp >= "now-30d/d"` ({pull}123590[#123590]).
+* Improves the exceptions interface by replacing the exceptions modal with a flyout ({pull}123408[#123408]).
+* Alert details flyout enhancements:
+** Shows different highlighted fields in an alert’s details flyout based on its type, category, and code ({pull}123239[#123239]).
+** Adds overview cards with key data to the alert details flyout ({pull}120347[#120347]).
+** Allows you to aggregate alert data based on any aggregatable ECS field instead of just 10 preset options ({pull}120610[#120610]).
```

Comment:
Just a thought -- I'm wondering if customers are going to know what `aggregatable ECS field` is supposed to really mean...

Reply:
@jmikell821 @benironside and I discussed alternatives for `aggregatable ECS field` and settled on not qualifying the type of ECS field that the **Count** and **Trend** tables use. In this particular case, specifying the type of ECS fields that the tables use might be more confusing than helpful. What do you think about changing that part of the summary to something more general like "a larger selection of fields"?

Reply:
Also @benironside the summary for #120610 might need to have its own line because it moreso affects the Trend and Count tables on the Alerts page and Overview page.